src/css/style.css

```diff
@@ -104,3 +109,80 @@ img {
 .card:hover {
     transform: scale(1.1)
 }
+
+
+
+/* code for service_cards page */
+
+.services_heading {
+display: flex;
+justify-content: center;
+font-weight: bolder;
+font-size: xx-large;
+margin-top: 40px;
+margin-bottom: 40px;
+}
+
+.servicesBox {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;}
+
+
+.servicesPicture {
+    width: 100%;
+    /* border-radius: 12px; */
+    height: 200px;
+    object-fit: cover;
+}
+
+.service_card_container {
+    width: 375px;
+    max-height: 600px;
+    margin: 15px;
+    margin-top: 10px;
+    transition: 0.2s;
+    padding: 10px;
+}
+
+.info_card {
```

Comment:
This can be combined with services_card_container. These styles don't need to be on their own element.
Will allow you to remove another div.